README.md

```diff
@@ -438,7 +440,3 @@ For security reasons, we don't want to use the default password for the default
 
 - **Q:** What versions of Istio, Knative, Cert-Manager, Argo, ... are compatible with Kubeflow? \
   **A:** Please refer to each individual component's documentation for a dependency compatibility range. For Istio, Knative, Dex, Cert-Manager and OIDC-AuthService, the versions in `common` are the ones we have validated.
```

Comment:
Here is a suggestion for the Kustomize 3.2 QA section.

```suggestion
  **A:** Please refer to each individual component's documentation for a dependency compatibility range. For Istio, Knative, Dex, Cert-Manager and OIDC-AuthService, the versions in `common` are the ones we have validated.
  
- **Q: ** How can I still use Kustomize version 3.2?
  **A: ** The only thing incompatible with Kustomize 3.2 is the `sortOptions` section in the `example/kustomization.yaml` file. I you want to use Kustomize 3.2 just comment out the entire section (`#` character in front of each line) and execute the one-liner. 
```

Reply:
@DomFleischmann I pushed a commit with a slightly refactored section for the earlier version of Kustomize. PTAL